### ISSUE TITLE
There is a problem with the construction of the WeatherMap class with __main__, causing some unexpected behavior

### ISSUE THREAD

**Issue by tim-fiola** [tim-fiola/network_traffic_modeler_py3#40]
2 weird issues:

1. The create_weathermap() call must be used within a script. If called at the live python3 CLI, the user will most likely receive an error message about not being able to find __main__
2. Calling create_weathermap() in a script causes the entire script to be run 2x

Example of item 1:
```
>>> from pyNTM import FlexModel
>>> from pyNTM import WeatherMap
>>> 
>>> model = FlexModel.load_model_file('model_test_topology_multidigraph.csv')
>>> 
>>> print("updating simulation manually")
updating simulation manually
>>> model.update_simulation()
Routing the LSPs . . . 
Routing 1 LSPs in parallel LSP group F-E; 1/3
Routing 1 LSPs in parallel LSP group A-F; 2/3
Routing 2 LSPs in parallel LSP group A-D; 3/3
LSPs routed (if present) in 0:00:00.001311; routing demands now . . .
Demands routed in 0:00:00.009475; validating model . . . 
>>> 
>>> vis = WeatherMap(model)
>>> 
>>> vis.create_weathermap()

*** NOTE: The make_visualization_beta function is a beta feature.  It may not have been as 
extensively tested as the pyNTM code in general.  The API calls for this may also 
change more rapidly than the general pyNTM code base.



Visualization is available at http://127.0.0.1:8050/


Dash is running on http://127.0.0.1:8050/

 * Serving Flask app "pyNTM.weathermap" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
/Users/tfi/Documents/python_crap/test_3.0/venv/bin/python3: can't find '__main__' module in '/Users/tfi/Documents/python_crap/test_3.0'
```

Example of item 2:
```
(venv) Timothys-Mini:test_3.0 timothyfiola$ python3 -i simple_test.py 
updating simulation manually
Routing the LSPs . . . 
Routing 1 LSPs in parallel LSP group F-E; 1/3
Routing 1 LSPs in parallel LSP group A-F; 2/3
Routing 2 LSPs in parallel LSP group A-D; 3/3
LSPs routed (if present) in 0:00:00.001266; routing demands now . . .
Demands routed in 0:00:00.009542; validating model . . . 
making visualization

*** NOTE: The make_visualization_beta function is a beta feature.  It may not have been as 
extensively tested as the pyNTM code in general.  The API calls for this may also 
change more rapidly than the general pyNTM code base.



Visualization is available at http://127.0.0.1:8050/


Dash is running on http://127.0.0.1:8050/

 * Serving Flask app "pyNTM.weathermap" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
updating simulation manually
Routing the LSPs . . . 
Routing 1 LSPs in parallel LSP group A-F; 1/3
Routing 2 LSPs in parallel LSP group A-D; 2/3
Routing 1 LSPs in parallel LSP group F-E; 3/3
LSPs routed (if present) in 0:00:00.001274; routing demands now . . .
Demands routed in 0:00:00.009616; validating model . . . 
making visualization

*** NOTE: The make_visualization_beta function is a beta feature.  It may not have been as 
extensively tested as the pyNTM code in general.  The API calls for this may also 
change more rapidly than the general pyNTM code base.



Visualization is available at http://127.0.0.1:8050/


```

**Comment by tim-fiola** [comment]
it looks like both of these problems are related to the  `debug=True` argument in `app.run_server(debug=True)` in the WeatherMap class.  Will confirm and issue a fix if so.

**Comment by tim-fiola** [comment]
fixed in 3.2
